impeller/entity/geometry/stroke_path_geometry.cc

```diff
@@ -192,7 +192,9 @@ StrokePathGeometry::CapProc StrokePathGeometry::GetCapProc(Cap stroke_cap) {
         vtx_builder.AppendVertex(vtx);
         vtx.position = position - orientation;
         vtx_builder.AppendVertex(vtx);
-        for (const auto& point : arc.CreatePolyline(scale)) {
+        std::vector<Point> arc_points;
```

Comment:
Or here?

Reply:
Same deal. This shouldn't typically result in a large number of points.